change/@fluentui-react-components-5b15aaa9-20f7-44b3-a369-8096b4cdf7eb.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 FluentUIV0 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=28530&commitId=504f803c0bdb62e78a9a449ee16964ff18738af6&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar.Avatar Example Image Shorthand.chromium.png|312| Changed|

</details>


  <details><summary><b>Card</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card.Card Example Centered.chromium.png|17465| Changed|

</details>


  <details><summary><b>Chat</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Chat.Message Reactions With Popup Teams High Contrast.chromium.png|263| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v0"/>